handlers/handlers.js

```diff
@@ -122,11 +122,20 @@ const deregisterHandler = async (client, msg) => {
   }
 };
 
+const chatHandler = (client, text) => {
+  fs.readFile(../chatbot/phraselist.txt, function(err, data){
+    var lines = data.split('\n');
+    sendMessage(client, lines[Math.floor(Math.random()*lines.length)]);
+ }
+
+};
+
 const handlerMap = {
   ".help": helpHandler,
   ".register": registerHandler,
   ".deregister": deregisterHandler,
   ".recognized-command": crawlRequestHandler,
+  "@gaming_bot": chatHandler,
```

Comment:
Overall, I'd prefer all commands that the gamingbot recognizes to begin with the same special character, for continuity.